lib/activity_log/log_class.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:altme/activity_log/activity_log.dart';
+import 'package:altme/app/app.dart';
```

Comment:
I don't think you need this import. I think I removed it from main while handling some chores

Reply:
yes.